yt_dlp/extractor/doodstream.py

```diff
@@ -52,7 +66,7 @@ def _real_extract(self, url):
         thumb = self._html_search_meta(['og:image', 'twitter:image'], webpage, default=None)
         token = self._html_search_regex(r'[?&]token=([a-z0-9]+)[&\']', webpage, 'token')
         description = self._html_search_meta(
-            ['og:description', 'description', 'twitter:description'], webpage, default=None)
+            ['og:description', 'description', 'twitter:description'], webpage, default=title)
```

Comment:
why?

Reply:
if there is no description it should be empty
```suggestion
            ['og:description', 'description', 'twitter:description'], webpage, default=None)
```